tests/comfy_cli/command/test_transfer_download.py

```
@@ -378,6 +396,248 @@ def test_ndjson_mode_emits_no_human_progress_line(self, fake_target, tmp_path, c
+        assert transfer._declared_content_length(self._Resp(value)) == expected
+
+
+class TestPartFileHelper:
+    """`_open_part_file` owns the raw fd end-to-end: any failure closes it and
+    removes the temp file, and a copy whose source vanishes must not leak the
+    descriptor either (the fd is wrapped before the source is opened)."""
+
+    def _open_fd_count(self):
+        return len(os.listdir("/proc/self/fd"))
+
+    def test_fchmod_failure_closes_fd_and_removes_temp(self, tmp_path, monkeypatch):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)